BoxContentSDK/BoxContentSDK/Clients/BOXContentClient.m

```diff
@@ -29,12 +29,15 @@
 NSString *const BOXDefaultOAuth2BaseURL = @"https://api.box.com/oauth2";
 NSString *const BOXDefaultAPIAuthBaseURL = @"https://account.box.com/api";
 NSString *const BOXDefaultAPIUploadBaseURL = @"https://upload.box.com/api/2.1";
+NSString *const BOXContentClientTempFolder = @"/TempContentClient";
```

Comment:
I don't think we should have the path delimiter in the constant... right? NSURL has methods for adding/removing/finding things in paths